src/App.tsx

```diff
@@ -42,6 +42,15 @@ const StyledProgressWrapper = styled.div`
   justify-content: center;
 `;
 
+const StyledMainContentLink = styled.a`
+  position: absolute !important;
+  overflow: hidden;
```

Comment:
Mener uu-tilsynet har kun hidden på denne så lenge den ikke har fokus.
Dvs du kan bruke shift + tab for å få den synlig. Men tror ikke det er viktig, jeg ser kun for meg at folk som bruker skjermlesere som ønsker å bruke den.

Reply:
la inn den også læll - også gjorde jeg om lenken som egen komponent, også pga mye css